Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocDateTimeValue.css

```
@@ -3,13 +3,11 @@
 span.bocDateTimeValue,
 span.bocDateValue
 {
+  min-height: var(--remotion-themed-min-height);
   width: 15em;
```

```suggestion
  width: 16em;
```

Please also update textvalue and multilinetextvalue for this.

In the end, we'll likely introduce a property for this. `--remotion-themed-control-width`. You can do it now if you want.